src/main/menu.js

```diff
@@ -142,14 +143,21 @@ if (isOSX()) {
   );
 }
 
+// Check for Updates...
+defaultTemplate[isOSX() ? 0 : 4].submenu.splice(isOSX() ? 1 : 0, 0, {
```

Comment:
Woah.. Don't mean to be a buzzkill.. but is there a nicer way to do this? This be crazy hard to read 😅